pkg/client/gen.go

```diff
@@ -67,6 +80,23 @@ func (*SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 		}
 	}
 
+	plugins := []*manifest.Manifest{}
+	if includes(cfg.Config.PluginSelections, e2ePluginName) {
+		plugins = append(plugins, e2eManifest(cfg))
+	}
+	if includes(cfg.Config.PluginSelections, systemdLogsName) {
```

Comment:
The 2x check on `includes` is obviously inefficient but:
1) the list of plugins is always short
2) this ensures a standard order: e2e, systemdlogs, other plugins (in followup PR)

Reply:
Why do these two have to come first and in that order?

Reply:
No hard reason it is exactly that order; it does make it easier for testing if we know the order, regardless of what it is. I wouldn't be against just alphabetizing them by plugin name though (which actually results in this same order for the default coincidentally)

Reply:
was just thinking it'd be great if the tests didn't rely on order and the e2es and systemdlog plugins were not considered special or blessed plugins

Reply:
Yea, that has been a matter of discussion lately and with some of the upcoming plugin changes that gets closer to achievable. The idea of it being alphabetically sorted would satisfy the testing need and also be compatible with that goal.